### DISCUSSION TITLE
Upgrading lima while VMs are Running

### DISCUSSION THREAD

**Discussion by mbentley** [lima-vm/lima#1900] [Q&A] [answered]
I was curious if anyone had any experience or suggestions regarding the upgrade process while VMs are running.  Is it safe to upgrade lima while I have VMs running or should I stop them before doing so?  I haven't specifically tried to upgrade while VMs are running but I wasn't sure if it would be an issue since I see a few processes running with `limactl` for the hostagent and user network.

**Comment by AkihiroSuda** [comment, marked as answer]
> upgrade lima while I have VMs running 

Discouraged, although it may work.
